configure.ac

```diff
@@ -94,19 +93,14 @@ AC_SUBST(COCKPIT_CFLAGS)
 AC_SUBST(COCKPIT_LIBS)
 
 # bridge with optional polkit
-AC_ARG_ENABLE(polkit, AC_HELP_STRING([--disable-polkit], [Disable usage of polkit]),,enable_polkit=$autodetect_polkit)
-AC_MSG_CHECKING([whether to build with polkit])
+AC_ARG_ENABLE(polkit, AC_HELP_STRING([--disable-polkit], [Disable usage of polkit]),,enable_polkit=yes)
+
 if test "$enable_polkit" = "yes"; then
-  AC_MSG_RESULT($enable_polkit)
-  if test "$autodetect_polkit" = "no"; then
-    AC_MSG_ERROR([Couldn't find polkit. Try installing polkit])
-  fi
+  PKG_CHECK_MODULES(POLKIT, [$POLKIT_REQUIREMENT])
   COCKPIT_BRIDGE_CFLAGS="$COCKPIT_CFLAGS $POLKIT_CFLAGS"
   COCKPIT_BRIDGE_LIBS="$COCKPIT_LIBS $POLKIT_LIBS"
   AC_DEFINE_UNQUOTED([WITH_POLKIT], [], [Build with polkit])
 else
-  enable_polkit="no"
-  AC_MSG_RESULT($enable_polkit)
   COCKPIT_BRIDGE_CFLAGS="$COCKPIT_CFLAGS"
```

Comment:
It seems a bit yucky to define this in a branch of a check that's polkit-specific, but that's something we can iron out later on...